archive/2021-01-04-DssSpell.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (C) 2020 Maker Ecosystem Growth Holdings, INC.
```

Comment:
It should get updated to 2020-2021 I guess

Reply:
This should also be modified in DssSpell.sol. Maybe better to get the original files reviewed before archiving them.